docs/reference-guides/data/data-core-editor.md

```diff
@@ -879,6 +879,45 @@ _Returns_
 
 -   `boolean`: Whether the post can be saved.
 
+### isEditorPanelEnabled
```

Comment:
Would be great if it explained a bit more what this editor panel is.

Reply:
it's just the panels that shows up in the sidebar of the editor: things like "post categories", "post featured image"...